COMP36212/content.tex

```diff
@@ -489,7 +487,15 @@ \subsubsection{Examples of Analysing Linear Multistep Methods}
 
 % Part Two
 
-\section{Optimisation and nature inspired algorithms}
+\section{Complex Networks}
+
+Eva's notes and lectures are really very good, use her notes, not mine :)
+
+% Discontinued section
+
+\section{Optimisation and nature inspired algorithms: DEPRECATED}
```

Comment:
If you use \section*{Optimisation...} (note the star) then it won't show up in the title page.